src/main/resources/default-lintConfig.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm fine either way - if you agree with the suggestions over there feel free to amend this config with those updates before you merge.  In either case we should get a few projects to try it out and discuss particulars in the next few weeks.